resolver/serviceResolver_test.go

```diff
@@ -2370,6 +2371,345 @@ func TestServiceResolver_SignMessage(t *testing.T) {
 	})
 }
 
+func TestServiceResolver_checkGuardianAndVerifyCode(t *testing.T) {
+	t.Parallel()
+	providedSender := "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"
+	providedRequest := requests.SetSecurityModeNoExpireMessage{
+		Code:       defaultFirstCode,
+		SecondCode: defaultSecondCode,
+		UserAddr:   providedSender,
+	}
+	t.Run("verify code should ErrInvalidGuardian", func(t *testing.T) {
```

Comment:
i think it's better here to provide explicitly an invalid guardian (from GetGuadianData call) as in the next examples

Reply:
updated